app/javascript/hyacinth_v1/graphql/dynamicFields.js

```diff
@@ -25,8 +25,8 @@ export const dynamicFieldPathQuery = gql`
       id
       displayLabel
       type: __typename
-      __typename # Apollo can't correctly cache if we override this field. Eventually we might override how cache identifiers are generated.
-      path {
+      __typename # Apollo can't correctly cache if we override this field. Maybe later: override how cache identifiers are generated.
```

Comment:
@barmintor I only updated this comment because I wanted to reduce the line length in advance of some eslint error cleanup.

Reply:
👍